pages/common/enscript.md

```diff
@@ -0,0 +1,23 @@
+# enscript
+
+> A tool to convert text files to PostScript, HTML, RTF, ANSI, and overstrikes.
+
+- Generate PostScript from a file and output to another:
+
+`enscript {{text_file}} --output={{output_file}}`
```

Comment:
Perhaps `{{path/to/input_file}}` and `{{path/to/output_file}}` for each example?

Reply:
Ok